### PR TITLE
HtmlService-example

### PR DESCRIPTION
**Pull request by IdoJS**

### CODE REVIEW

File: packages/examples/package.json [hunk 1]
```diff
@@ -8,11 +8,11 @@
   "files": [
     "dist",
     "lib",
-    "es",
-    "src"
+    "es"
   ],
   "license": "MIT",
   "scripts": {
+    "clean": "rm -rf node_modules",
```
Comment (dmitriyslaym) [inline packages/examples/package.json]: I think, this script should also remove all the builds

File: packages/scalecube-microservice/src/Routers/AB.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+// import { lookUp } from '../Microservices/Registry';
```
Comment (dmitriyslaym) [inline packages/scalecube-microservice/src/Routers/AB.ts]: do we need it now?

File: packages/scalecube-microservice/src/index.ts [hunk 3]
```diff
@@ -1,2 +1,5 @@
-export { Microservices } from './Microservices/Microservices';
-export { ASYNC_MODEL_TYPES } from './helpers/constants';
+import { Microservices } from './Microservices/Microservices';
+import { ASYNC_MODEL_TYPES } from './helpers/constants';
+
+export { Microservices, ASYNC_MODEL_TYPES };
+export default Microservices;
```
Comment (dmitriyslaym) [inline packages/scalecube-microservice/src/index.ts]: don't like this approach, have described above why. Is there any special reason to to it like this?

File: packages/scalecube-microservice/tests/mocks/globals.ts [hunk 4]
```diff
@@ -0,0 +1,5 @@
+// @ts-ignore
```
Comment (dmitriyslaym) [inline packages/scalecube-microservice/tests/mocks/globals.ts]: don't need this file at all anymore

File: packages/scalecube-microservice/tsconfig.json [hunk 5]
```diff
@@ -1,14 +1,16 @@
...
     "lib": ["dom", "es2015", "es2017.object", "es5", "es6", "es2016"],
     "declaration": true,
+    "outDir": "./lib",
     "strict": true,
-    "noUnusedLocals": true,
     "moduleResolution": "node",
+    "baseUrl": "./src",
     "esModuleInterop": true,
-    "baseUrl": "./src"
+    "experimentalDecorators": true
```
Comment (dmitriyslaym) [inline packages/scalecube-microservice/tsconfig.json]: are you using decorators here?

File: packages/examples/src/html-index.js [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+        }
+      ]
+    }).createProxy({
+      serviceDefinition: definition
+    });
+
+    proxy.render();
+
+
+  })(window.sc.default, window.sc.ASYNC_MODEL_TYPES);
```
[the remark this answers is not in the document]
Comment (IdoJS) [reply packages/examples/src/html-index.js]: you can access it as window.sc.default or window.sc.Microservices

File: packages/examples/package.json [hunk 7]
```diff
@@ -34,6 +31,6 @@
     "typescript": "^3.2.4"
   },
   "dependencies": {
-    "@scalecube/scalecube-microservice": "^0.1.3-alpha.4"
+    "@scalecube/scalecube-microservice": "snapshot"
```
Comment (dmitriyslaym) [inline packages/examples/package.json]: I think we also have to use the same version of the package, that is specified in local package.json of @scalecube/scalecube-microservice

File: packages/scalecube-discovery/package.json [hunk 8]
```diff
@@ -2,23 +2,25 @@
   "name": "@scalecube/scalecube-discovery",
   "version": "0.1.1-alpha.0",
   "private": false,
-  "license": "MIT",
   "main": "lib/index.js",
-  "unpkg": "dist/index.js",
-  "module": "es/index.js",
+  "unpkg": "dist-min/index.js",
+  "module": "lib/index.js",
```
Comment (dmitriyslaym) [inline packages/scalecube-discovery/package.json]: I think es module should be used here (so "es/index.js")

File: packages/scalecube-discovery/package.json [hunk 9]
```diff
@@ -2,23 +2,25 @@
...
     "dist",
     "lib",
     "es"
   ],
+  "license": "MIT",
   "scripts": {
-    "build": "rm -rf .cache && yarn run build:lib && yarn run build:dist && yarn run build:es",
-    "build:lib": "rm -rf lib && tsc --module commonJS --target es5 --outDir lib",
-    "build:dist": "rm -rf dist && parcel build src/index.ts --experimental-scope-hoisting --out-dir dist --global sc",
+    "clean": "rm -rf node_modules && rm -rf lib && rm -rf es && rm -rf dist && rm -rf dist-min",
```
Comment (dmitriyslaym) [inline packages/scalecube-discovery/package.json]: suggest to also clean .cache here

File: packages/scalecube-microservice/package.json [hunk 10]
```diff
@@ -2,24 +2,25 @@
...
     "dist",
     "lib",
     "es"
   ],
   "license": "MIT",
   "scripts": {
-    "build": "rm -rf .cache && yarn run build:lib && yarn run build:dist && yarn run build:es",
-    "build:lib": "rm -rf lib && tsc --module commonJS --target es5 --outDir lib",
-    "build:dist": "rm -rf dist && parcel build src/index.ts --experimental-scope-hoisting --out-dir dist --global sc",
+    "clean": "rm -rf node_modules && rm -rf lib && rm -rf es && rm -rf dist && rm -rf dist-min",
```
Comment (dmitriyslaym) [inline packages/scalecube-microservice/package.json]: include .cache

**Pull request merged**